client/simple.go

```diff
@@ -80,7 +99,7 @@ func (c *SimpleCommitmentClient) GetData(ctx context.Context, comm []byte) ([]by
 	}
 
 	if resp.StatusCode != http.StatusOK {
-		return nil, fmt.Errorf("received error response, code=%d, msg = %s", resp.StatusCode, string(b))
+		return nil, fmt.Errorf("received error response when reading from eigenda, code=%d, msg = %s", resp.StatusCode, string(b))
```

Comment:
we don't know that the error comes from eigenda right? it could come from the proxy itself, like some 500 or whatever

Reply:
maybe just change to "from eigenda-proxy"?

Reply:
[a2e80ee](https://github.com/Layr-Labs/eigenda-proxy/pull/209/commits/a2e80ee0dd07827280a4383a2753acb8f4e4169e)